rule_packages/cpp/Conversions2.json

```diff
@@ -0,0 +1,128 @@
+{
+  "MISRA-C++-2023": {
+    "RULE-8-2-1": {
+      "properties": {
+        "enforcement": "decidable",
+        "obligation": "required"
+      },
+      "queries": [
+        {
+          "description": "Casting from a virtual base class to a derived class using anything other than dynamic_cast causes undefined behavior.",
+          "kind": "problem",
+          "name": "A virtual base class shall only be cast to a derived class by means of dynamic_cast",
+          "precision": "very-high",
+          "severity": "error",
+          "short_name": "VirtualBaseClassCastToDerived",
+          "tags": [
+            "scope/single-translation-unit"
```

Comment:
Also, almost forgot to add that these need additional metadata tags (correctness, maintainability, etc)